### PR TITLE
Massive Church Update (check wiki too)

### PR DESCRIPTION
**Pull request by Kazkin**
-Added a church lunchbox to kitchen vendors.
-Added a lemniscate lunchbox that can be obtained through a litany, comes preloaded with 2 omni pretzals, 2 precept sandwiches, and 2 cahors classic cans.
-Potentially fixed an issue with claymores returning plasteel when crafted with steel.
-Deleted old lunchbox code as other stuff was written that works more cleanly.
-Added 2 new church mods for factorial, obtainable by their litanies at an altar. Overclocked sanctifier and and factorial excruciator hyper lens. They got +50% benefits AND penalties that the regular variants get.
-Fixed a wide selection of typos.
-Added lunchboxes to load outs.
-Cruciform upgrades can now be placed on altars the same way one does a cruciform.
-Soul hunger now deals 10 toxins per use (up from 5).
-Cruciform upgrades now no longer require getting naked to install them. (Not tested, but should work).
-Canticle of absolution is now an apostle only litany, it cures clone damage and removes mutations.
-ALL PATHS NOW GAIN 2 ADDITIONAL LITANIES
--Tesellate
-Realignment, when cast it cures toxins, liver damage, and clears toxic chemicals/stims from the blood while curing addictions. Slows down the person its used on while it works.
-Matyr's Calculation, when cast it heals for a massive amount, but can only be used every half hour and puts the speaker to sleep for 30 seconds as it overwhelms them.
--Lemniscate
-Food for the Masses, 60 minute cooldown, spawns a lunchbox filled with food. Pretzal have what is effectively tricord, sandwiches give bonus rob/tgh and have hyperzine as well as lots of protien. Cahors classic has 45 cahors and 15 spaceacillin variant.
-Infinite Hymn, gain a self speed boost for 5 minutes with a 10 minute cooldown, gives the same boost as hyperzine without the side effects. **Does not stack with hyperzine.**
--Monomial
-Inner Peace, if you have 25 or more inspiration, you can gift double the amount to someone else (50). Does not work when used by synthetics or on synthetics. Incredibly good for other people who need inspiration (such as psions/nanogaters) or monomials that don't care for getting inspired. Has a 10 minute cooldown to prevent someone from spamming it to give someone else 200 inspiration.
-Bulwark of Harmony, this litany last 1 minute and gives you -50% damage reduction from brute/burn/toxin/oxygen but reduces your speed by 300%, effectively making you a slow moving mountain that can tank alot of damage. Has a 30 minute cooldown.
--Divisor
-Probability Coefficient, an improved version of reveal advisories that has no failure chance, never misleads you, and gives you multiple types of hazards information (traps/mobs/carrion/etc,.). It also tells you if simple mobs and superior mobs are nearby, which can be very useful if you know the terrain.
-Divine smite, a litany I highly advise hotkeying. It last only 5 seconds and has a cooldown of 5 minutes, but it increases your melee damage by 20% while active.
--Factorial
-Prayer of the Forge, 60 minute CD, can get a new testament or old testament disk from an altar to use in any autolathe or biolathe. Still requires finding materials.
-Hymn of the Engineer, 120 minute CD, gives you a choice of 1 of 5 cruciform upgrades or 2 suped up tool mods.
-Added some new cult chems specifically for their food and litanies.

### CODE REVIEW

File: code/modules/core_implant/cruciform/rituals/path.dm [hunk 1]
```diff
@@ -540,3 +741,147 @@
...
+			new /obj/item/computer_hardware/hard_drive/portable/design/nt_old_guns(altar.loc)
+			if(user.species?.reagent_tag != IS_SYNTHETIC)
+				if(user.nutrition >= nutri_cost)
+					user.nutrition -= nutri_cost
+				else
+					to_chat(user, SPAN_WARNING("You manage to cast the litany at a cost. The physical body consumes itself..."))
+					user.vessel.remove_reagent("blood",blood_cost)
+			return TRUE
+		if (response == "Cancel Litany")
+			fail("You decide not to obtain a prosthetic at this time.", user, C)
```
Comment (r4d6) [inline code/modules/core_implant/cruciform/rituals/path.dm:785]: Mention prosthetics despite being a disk.

File: code/modules/core_implant/cruciform/rituals/path.dm [hunk 2]
```diff
@@ -540,3 +741,147 @@
...
+
+	var/obj/machinery/optable/altar = locate(/obj/machinery/optable/altar) in OBJS
+
+	if(!altar)
+		fail("This is not an altar, the litany is useless.", user, C)
+		return FALSE
+
+	if(altar)
+		var/response = input(user, "Which upgrade do you require?") in list("Excrutiator Hyper Lens", "Overclocked Sanctifier", "Nature's Blessing", "Cleansing Presence", "Faith's Shield", "Martyr's Gift", "Wrath of God", "Speed of the Chosen", "Cancel Litany")
+		if (response == "Excrutiator Hyper Lens")
```
Comment (r4d6) [inline code/modules/core_implant/cruciform/rituals/path.dm:811]: Here a switch statement would be better for optimization.

File: code/modules/core_implant/cruciform/rituals/path.dm [hunk 3]
```diff
@@ -540,3 +741,147 @@
...
+	var/list/OBJS = get_front(user)
+
+	var/obj/machinery/optable/altar = locate(/obj/machinery/optable/altar) in OBJS
+
+	if(!altar)
+		fail("This is not an altar, the litany is useless.", user, C)
+		return FALSE
+
+	if(altar)
+		var/response = input(user, "Which upgrade do you require?") in list("New Testament Arms Disk", "Old Testament Arms Disk", "Cancel Litany")
```
Comment (r4d6) [inline code/modules/core_implant/cruciform/rituals/path.dm:765]: Switch statement would be better

File: code/modules/reagents/reagents/medicine.dm [hunk 4]
```diff
@@ -418,10 +413,42 @@
...
+
+/datum/reagent/medicine/spaceacillin/holy
+	name = "Holycilin"
+	id = "holycilin"
+	description = "A chemical of unknown origin, believed to be derived from cahors and spaceacillin that functions identical to the latter."
+	taste_description = "sweetness"
+	appear_in_default_catalog = FALSE
+	constant_metabolism = TRUE
+	scannable = 0
+	overdose = 0
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/reagents/medicine.dm:450]: You think this should be ODing at 0?

Comment (Kazkin) [reply code/modules/reagents/reagents/medicine.dm:450]: Setting it to 0 should stop it from causing OD at all.

**Pull request merged**